Exercise_B/index.html

```diff
@@ -18,17 +19,40 @@
   </head>
 
   <body>
-    <!--Header (flower shop logo block element)-->
-    <!--Banner Jumbotron of the image-->
-
-    <!--Four containers with img / text-->
-    <!-- 4 containers -->
-
-    <!-- Payments Div -->
-    <!-- Consultation Div -->
+    <div class='header'>
+      <img src="./assets/images/logo.png" alt="">
+      
+    </div>
 
-    <!-- Delivery Div -->
+    <div class = 'mid'>
+      <p>Beautiful Flowers</p>
+      <a href="">Learn more</a>
+    </div>
+    <div class='bottom'>
+      <div class='container'>
+        <div class='row'>
+          <div class = 'col-12 col-lg-3 col-md-6'>
+            <img src="./assets/images/flower-icon.png" alt="">
+            <p id = 'onee'>Flowers for every kind of situation</p>
+          </div>
+          <div class = 'col-12 col-lg-3 col-md-6'>
+            <img src="./assets/images/payment-icon.png" alt="">
+            <p>All major payments kinds accepted</p>
+          </div>
+          <div class = 'col-12 col-lg-3 col-md-6' id ='three'>
+            <img src="./assets/images/phone-icon.png" alt="">
+            <p>Free consulation</p>
+          </div>
+          <div class = 'col-12 col-lg-3 col-md-6' id = 'four'>
+            <img src="./assets/images/\/truck-icon.png" alt="">
+            <p>City wide delivery</p>
+          </div>
+        </div>
+      </div>
+    </div>
 

```

Comment:
Fantastic implementation of the bootstrap row and column system.
The only thing I would suggest altering is your use two encapsulating divs around your row container. 
You only really need one layer if anything,